src/app/Plans/components/PlanDetails.tsx

```diff
@@ -59,143 +63,173 @@ const PlanDetails: React.FunctionComponent<IPlanDetailsProps> = ({
+              mappingType={MappingType.Network}
+              mapping={networkMapping}
+              isLabel={false}
+              disableOk={true}
+            />
+          ) : null}
+        </DescriptionListTerm>
+        <DescriptionListDescription
+          id="details-network-mapping"
+          aria-labelledby="network-mapping-label"
+        >
+          <MappingDetailView
+            mappingType={MappingType.Network}
+            sourceProviderType={sourceProvider?.type || 'vsphere'}
+            mapping={networkMapping}
+          />
+        </DescriptionListDescription>
+      </DescriptionListGroup>
+      <DescriptionListGroup>
+        <DescriptionListTerm id="storage-mapping-label">
+          Storage mapping{' '}
+          {storageMapping ? (
+            <MappingStatus
+              className={spacing.mlXs}
+              mappingType={MappingType.Storage}
+              mapping={storageMapping}
+              isLabel={false}
+              disableOk={true}
+            />
+          ) : null}
+        </DescriptionListTerm>
+        <DescriptionListDescription
+          id="details-storage-mapping"
+          aria-labelledby="storage-mapping-label"
+        >
+          <MappingDetailView
+            mappingType={MappingType.Storage}
+            sourceProviderType={sourceProvider?.type || 'vsphere'}
+            mapping={storageMapping}
+          />
+        </DescriptionListDescription>
+      </DescriptionListGroup>
+      <DescriptionListGroup>
+        <DescriptionListTerm id="migration-type-label">
+          Migration type{' '}
+          {warmCriticalConcernsFound.length > 0 ? (
+            <Popover
+              hasAutoWidth
+              bodyContent={
+                <>
+                  <Text>
+                    Warm migration will fail for one or more VMs because of the following
+                    conditions:
+                  </Text>
+                  <List className={`${spacing.mtSm} ${spacing.mlMd}`}>
+                    {warmCriticalConcernsFound.map((label) => (
+                      <ListItem key={label}>{label}</ListItem>
+                    ))}
+                  </List>
+                </>
+              }
+            >
+              <Button variant="link" isInline>
+                <StatusIcon status="Error" className={spacing.mlXs} />
+              </Button>
+            </Popover>
+          ) : null}
+        </DescriptionListTerm>
+        <DescriptionListDescription
+          id="details-migration-type"
+          aria-labelledby="migration-type-label"
+        >
+          {plan.spec.warm ? 'Warm' : 'Cold'}
+        </DescriptionListDescription>
+      </DescriptionListGroup>
+      <DescriptionListGroup>
+        <DescriptionListTerm id="migration-hooks-label">Hooks</DescriptionListTerm>
+        <DescriptionListDescription id="details-plan-hooks" aria-labelledby="migration-hooks-label">
+          {hooksDetails && hooksDetails?.length > 0 ? (
             <Grid>
```

Comment:
I wonder if this hooks grid should also itself be a description list? i guess it probably doesn't matter. I'll merge anyway for now.